packages/pg-v5/commands/pull.js

```
@@ -125,19 +112,69 @@ const maybeTunnel = function * (herokuDb) {
   return herokuDb
 }
 
+function spawnPipe (commandOne, commandTwo) {
+  return new Promise((resolve, reject) => {
+    commandOne.stdout.on('data', (data) => {
+      commandTwo.stdin.write(data)
+    })
+
+    commandOne.stderr.on('data', (data) => {
+      console.log(`${data.toString().trim()}`)
+    })
+
+    commandOne.on('close', (code) => {
+      if (code !== 0) {
+        cli.exit(code)
```

it's better to throw an error like `psql dump errored with ${code}`